docs/user/SUMMARY.md

```diff
@@ -2,7 +2,12 @@
 
 # User Guides
 
-- [Adding Glean to your project](user/adding-glean-to-your-project.md)
+- [Adding Glean to your project](user/adding-glean-to-your-project/index.md)
+  - [C#](user/adding-glean-to-your-project/csharp.md)
+  - [Javascript](user/adding-glean-to-your-project/javascript.md)
+  - [Kotlin](user/adding-glean-to-your-project/kotlin.md)
+  - [Python](user/adding-glean-to-your-project/python.md)
+  - [Swift](user/adding-glean-to-your-project/swift.md)
```

Comment:
nit: Ok to keep the order we use everywhere else? Kotlin, Swift, Python, Rust, Javascript, Firefox Desktop.

Also, would you mind filing follow ups for the missing pages?

Reply:
Ok, i guess: but why was that order picked? Why not alphabetical?

Reply:
I just followed the prevalent order that I saw in the book before. Alphabetical might have been a better option you are right. I am thinking the logic behind that ordering is the order in which things were implemented + added to the book.

Reply:
https://bugzilla.mozilla.org/show_bug.cgi?id=1713573